crates/shared/src/tenderly_api.rs

```diff
@@ -102,6 +102,24 @@ impl TenderlyApi for TenderlyHttpApi {
         Ok(serde_json::from_str(&body)?)
     }
 
+    fn log(&self, simulation: SimulationRequest) -> Result<()> {
+        let request_url = crate::url::join(&self.api, "simulate");
+        let simulation_url =
+            crate::url::join(&self.dashboard, "simulator/$SIMULATION_ID").to_string();
+        let body = serde_json::to_string(&simulation)?;
+
+        #[rustfmt::skip]
+        tracing::error!(
```

Comment:
Since this is going to be logged for both `err` and `ok` cases, why do we need it on the `error` level?

Reply:
Oops, should be `trace` or `debug`. During development I tend to emit new logs as `error` because that is easy to see in our see of logs.
Good catch!